src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/TensorPrimitives.netcore.cs

```diff
@@ -1404,22 +1404,25 @@ private static Vector512<float> FusedMultiplyAdd(Vector512<float> x, Vector512<f
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private static unsafe Vector128<float> LoadRemainderMaskSingleVector128(int validItems) =>
-            Vector128.LoadUnsafe(
-                ref Unsafe.As<uint, float>(ref MemoryMarshal.GetReference(RemainderUInt32Mask_16x16)),
-                (uint)((validItems * 16) + 12)); // last four floats in the row
+            Vector128.ConditionalSelect(
+                Vector128.LessThan(Vector128.Create(3, 2, 1, 0), Vector128.Create(validItems)).AsSingle(),
+                Vector128<float>.AllBitsSet,
+                Vector128<float>.Zero);
```

Comment:
This is going to be slower and result in a 128-bit constant (or more for other sizes) emitted at runtime per method this gets inlined into.

I think the table is ultimately better and we can get rid of it longer term using a proper JIT intrinsic.

-----------

If we really don't want the table, then achieving this with just a broadcast + comparison should be sufficient, since `LessThan` already produces a per-element mask. You can use `GreaterThanOrEqual` if you need the mask inverted

Reply:
Ok, I'll stick with the table. I was on the fence anyway. Just don't like its bulkiness.

Reply:
That is, since `TrailingMask` wants to handle only trailing elements, it will skip processing the first `n` items that have already been processed.

So if `2` items remain, you have `3 < 2, 2 < 2, 1 < 2, 0 < 2` which produces `Zero, Zero, AllBitsSet, AllBitsSet` already